### ISSUE TITLE
how to use the RGBD superversied on nerf?

### ISSUE THREAD

**Issue by silver-obelisk** [Markus-Pobitzer/RGBD-NeRF#1]
Thanks for the excellent work！I am also researchring on how to use the RGBD superversied on nerf.  I read the code of DS-nerf ,but I found load_sensor_depth on load_llff.py actually didnot work，just like https://github.com/dunbar12138/DSNeRF/issues/14#issue-1001083662 .
So I wonder what should we change compaired with code of DS_nerf?  Could you provid us a tutorial so we can use RGBD-supervised loss in our project? 
Besides,I wonder  what is simplices.npy use for? I cannot find the same file on DS-nerf.

**Comment by Markus-Pobitzer** [comment]
In my project, I used synthetic depth maps (i.e. acquired by a NeRF-network and not by a sensor). This makes some things more manageable.

The depth maps I used have a range of values in [min_depth, max_depth]; therefore, loading them into the pipeline was straightforward. Have a look at this [line](https://github.com/Markus-Pobitzer/RGBD-NeRF/blob/main/load_blender.py#L79) of my load_blender.py file to bring them in the right format. For the llff data, it is similar. 

Make sure you arrange them in the right order in your data loader, see [link](https://github.com/Markus-Pobitzer/RGBD-NeRF/blob/e4e05500e2385df12aab60859c9e48dd6c219d58/run_nerf.py#L1021-L1023) for my implementation for the use_batching=True case.

In the training loop extract the depth tensors [link](https://github.com/Markus-Pobitzer/RGBD-NeRF/blob/e4e05500e2385df12aab60859c9e48dd6c219d58/run_nerf.py#L1123) and use it for the computation of the loss [here](https://github.com/Markus-Pobitzer/RGBD-NeRF/blob/e4e05500e2385df12aab60859c9e48dd6c219d58/run_nerf.py#L1251).

Not sure where you found simplices.npy?
